src/test/java/it/gov/pagopa/payment/repository/TransactionInProgressRepositoryExtImplTest.java

```diff
@@ -159,7 +162,7 @@ void updateTrxAuthorized() {
         transaction.setUserId("USERID%d".formatted(1));
         transactionInProgressRepository.save(transaction);
 
-        transactionInProgressRepository.updateTrxAuthorized(transaction, reward, List.of(), CommonPaymentUtilities.getInitiativeRejectionReason(transaction.getInitiativeId(), List.of()));
+        transactionInProgressRepository.updateTrxAuthorized(transaction, new AuthPaymentDTO(), CommonPaymentUtilities.getInitiativeRejectionReason(transaction.getInitiativeId(), List.of()));
```

Comment:
there is something wrong here...
you are not filling the AuthPaymentDTO, while you are supposed to fill it and check that the result has correctly read the expected info